From 8ee42c56f1f5f959ce1c66ad4b33235a0b69c774 Mon Sep 17 00:00:00 2001
From: Bernd Kuhls <bernd@kuhls.net>
Date: Sat, 17 May 2025 12:19:09 +0200
Subject: [PATCH] Fix build with gcc-15

Upstream: not applicable

The code which included uint16_t was removed in the to-be-released
Piers branch:
https://github.com/xbmc/inputstream.adaptive/commit/2afe7b8801d2da3059bfb2df0e5a6981fb9e46e9#diff-3dcaf13e4dbb2a79fa703382b7469e3c4ab1f0994928a5c0fca2627b8a9c8eac

Signed-off-by: Bernd Kuhls <bernd@kuhls.net>
---
 src/common/AdaptiveUtils.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/common/AdaptiveUtils.h b/src/common/AdaptiveUtils.h
index 7563cb86..89812532 100644
--- a/src/common/AdaptiveUtils.h
+++ b/src/common/AdaptiveUtils.h
@@ -13,6 +13,7 @@
 #include <memory>
 #include <string_view>
 #include <vector>
+#include <cstdint>
 
 // forwards
 class AP4_Movie;
-- 
2.39.5

